scripts/Obtain-Workspace.ps1

```diff
@@ -170,19 +164,23 @@ Write-Host
 
 Create-ComponentConfigJson($epoch_secs)
 
-Write-Host
-Write-Host "Checking for old resource groups"
-Write-Host
 
-$old_rg_list =Get-OldResourceGroups($epoch_secs-2*$window_seconds)
-if( $old_rg_list.count -gt 0)
-{
-    Write-Host "Found $($old_rg_list.count) resource groups to clean up"
-    foreach( $rg in $old_rg_list)
-    {
-        Write-Host "Cleaning up $($rg.name)"
-        az group delete --name $rg.name --yes
+
+Write-Host
+if ( $cullWorkspaces ) {
+    Write-Host "Checking for old resource groups ($cullWorkspaces)"
+    Write-Host 
+    $old_rg_list = Get-OldResourceGroups($epoch_secs - 2 * $window_seconds)
+    if ( $old_rg_list.count -gt 0) {
+        Write-Host "Found $($old_rg_list.count) resource groups to clean up"
+        foreach ( $rg in $old_rg_list) {
+            Write-Host "Cleaning up $($rg.name)"
+            az group delete --name $rg.name --yes
```

Comment:
> az group delete --name $rg.name --yes [](http://example.com/codeflow?start=11&length=38)

Do we make sure there aren't any existing builds/runs in progress before we do this?

Reply:
Any build which is using this will have been running for over `windowSeconds`. Which defaults to a day.